whelk-core/src/main/groovy/whelk/JsonLd.groovy

```diff
@@ -642,6 +663,12 @@ class JsonLd {
 
     //==== Class-hierarchies ====
 
+    Set<String> getSuperClasses(String type) {
```

Comment:
In general more useful to have this as an ordered List?
Convert to Set at use instead?